artifactory/services/utils/artifactoryutils.go

```diff
@@ -232,7 +232,20 @@ func getLatestBuildNumberFromArtifactory(buildName, buildNumber, projectKey stri
 		buildRepo = projectKey
 	}
 	buildRepo += buildRepositoriesSuffix
-	aqlBody := CreateAqlQueryForLatestCreated(buildRepo, buildName)
+	buildNameTrimDelimiter := strings.TrimSuffix(buildName, "/")    // "buildname/" >> "buildname" (remove trailing '/' if exist and there is no build name)
```

Comment:
I'm not sure that "... there is no build name" is quite clear. Maybe we should better expalin this case.